### ISSUE TITLE
Resuming uploads no longer works due to missing metadata

### ISSUE THREAD

**Issue by nellh** [OpenNeuroOrg/openneuro#167]
Getting a client error resuming a failed upload.

```
Uncaught TypeError: Cannot read property 'datasets' of undefined
    at Object._onFileSelect [as onChange] (dataset.jsx:419)
    at Upload._onFileSelect (file-select.jsx:84)
    at HTMLUnknownElement.callCallback (react-dom.development.js:1299)
    at Object.invokeGuardedCallbackDev (react-dom.development.js:1338)
    at Object.invokeGuardedCallback (react-dom.development.js:1195)
    at Object.invokeGuardedCallbackAndCatchFirstError (react-dom.development.js:1209)
    at executeDispatch (react-dom.development.js:1432)
    at Object.executeDispatchesInOrder (react-dom.development.js:1454)
    at executeDispatchesAndRelease (react-dom.development.js:1969)
    at executeDispatchesAndReleaseTopLevel (react-dom.development.js:1980)
    at Array.forEach (<anonymous>)
    at forEachAccumulated (react-dom.development.js:1946)
    at Object.processEventQueue (react-dom.development.js:2139)
    at runEventQueueInBatch (react-dom.development.js:2151)
    at Object.handleTopLevel [as _handleTopLevel] (react-dom.development.js:2161)
    at handleTopLevelImpl (react-dom.development.js:1800)
    at batchedUpdates (react-dom.development.js:13238)
    at performFiberBatchedUpdates (react-dom.development.js:1646)
    at stackBatchedUpdates (react-dom.development.js:1637)
    at batchedUpdates (react-dom.development.js:1651)
    at Object.batchedUpdatesWithControlledComponents [as batchedUpdates] (react-dom.development.js:1664)
    at dispatchEvent (react-dom.development.js:1874)
```

**Comment by nellh** [comment]
Turns out this is a regression caused by the fix for #6. Working out a way to do this without requesting all metadata before starting an upload.

**Comment by chrisgorgo** [comment]
Related/duplicate https://github.com/OpenNeuroOrg/openneuro/issues/140
